backend/src/db/migrations/20230923060705_add_surveys_steps_table.ts

```diff
@@ -0,0 +1,36 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'surveys';
+
+const ColumnName = {
+  FEELINGS: 'feelings',
+  GOALS: 'goals',
+  WORRIES: 'worries',
+  MEDITATION_EXPERIENCE: 'meditation_experience',
+  JOURNALING_EXPERIENCE: 'journaling_experience',
+} as const;
+
+const POSTGRE_ARRAY_TYPE = 'varchar(1000)[]';
+const POSTGRE_STRING_TYPE = 'varchar(255)';
```

Comment:
specify them where needed
```suggestion
```

Reply:
<img width="403" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/90299049/7387e3dd-3896-4066-9605-f01bd5048432">

How I could avoid this?

Reply:
It's needed specification, that was about another one